nextflow_schema.json

```diff
@@ -78,6 +78,14 @@
                     "description": "Full path to GFF annotation file.",
                     "fa_icon": "fas fa-file-invoice"
                 },
+                "additional_annot": {
+                    "type": "string",
+                    "format": "file-path",
+                    "mimetype": "text/plain",
+                    "pattern": "^\\S+(\\.gff|\\.gtf)(\\.gz)?$",
+                    "description": "Full path to additional annotation file in GTF or GFF format.",
```

Comment:
You could mention the optional `.gz`.
I assume there's no need to decompress in such case?

Reply:
It is the same format as the [--gff param](https://github.com/nf-core/viralrecon/blob/1883a68346f87cc2534323f8dbeceec554a7edc2/nextflow_schema.json#L74)

Should I add the .gz in the description for both? Both of them have a `GUNZIP_GFF` process in the case they are compressed.